inbox/helpers_test.go

```diff
@@ -25,7 +26,11 @@ func TestGetGroupAndAssociatedMemberIdentifier(t *testing.T) {
 		},
 	}
 
-	results, err := getGroupAndAssociatedMemberIdentifier(jamesIdentifier, nil)
+	expectedParent := make(map[string]string)
+	expectedParent["database"] = "namespace"
+	expectedParent["namespace"] = "cluster"
+
```

Comment:
expecteds should be on full form of the output

Reply:
Updated